ShowCase : Ajax framework basic : KeyUp listener blows

### Describe the bug

Go to https://www.primefaces.org/showcase/ui/ajax/basic.xhtml
- Focus on "Input" field
- Hit Tab to go to the next field "KeyUp"
The page blows and you'll get : 
```
Caused by: java.lang.NullPointerException
        at deployment.primefaces-showcase-14.0.0-SNAPSHOT.war//org.primefaces.showcase.view.ajax.BasicView.handleKeyEvent(BasicView.java:84)
```

### Reproducer

_No response_

### Expected behavior

You must be able to reach the field "KeyUp" with the Tab key.

Note : it works if you clic on the "KeyUp" field.

### PrimeFaces edition

Community

### PrimeFaces version

14.0.0-SNAPSHOT

### Theme

_No response_

### JSF implementation

Mojarra

### JSF version

2.3

### Java version

17

### Browser(s)

_No response_

--- a/primefaces-showcase/src/main/java/org/primefaces/showcase/view/ajax/BasicView.java
+++ b/primefaces-showcase/src/main/java/org/primefaces/showcase/view/ajax/BasicView.java
@@ -81,7 +81,9 @@ public class BasicView implements Serializable {
     }
 
     public void handleKeyEvent() {
-        text5 = text5.toUpperCase();
+        if (text5 != null) {
+            text5 = text5.toUpperCase();
+        }
     }
 
     public String getText1() {

